### DISCUSSION TITLE
[Feature Clarification] Storage of Test Metrics in External Databases (Postgres/Delta Tables) in Great Expectations v1.0.6

### DISCUSSION THREAD

**Discussion by MidnightRambo** [great-expectations/great_expectations#10589] [Support]
**Background:**
We have recently upgraded from Great Expectations version 0.18 to 1.0.6, marking a transition from a pre-release to a stable version. In version 0.18, it was possible to create a metrics store, allowing us to save metrics and test results in a Postgres SQL server. This functionality was integral to our workflow, enabling persistent storage and easy access to test outcomes.

**Issue:**
Upon upgrading to version 1.0.6, we’ve encountered challenges related to the metrics store:

- Documentation Redirect:
The official Great Expectations documentation, specifically the “[Configure a metrics store](https://docs.greatexpectations.io/docs/0.18/oss/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore/)” manual, redirects to the introduction page for version 1.0.6, making it unclear whether the metrics store feature is still supported or if its configuration has changed.

- Unclear Feature Availability:
Given the lack of accessible documentation for the metrics store in the new version, we are uncertain whether:

1. The metrics store feature has been deprecated or removed.
2. There is an alternative method to achieve the same functionality.
3. External storage of metrics is supported in a different manner.

- Related Discussion:
In Discussion [#10376](https://github.com/great-expectations/great_expectations/discussions/10376), a user inquired about writing checkpoint results to Delta Tables using PySpark on Databricks. This raises the question of whether Great Expectations v1.0.6 supports storing results in table formats like Delta or Postgres.

**Request for Clarification:**
Is the metrics store feature still available in Great Expectations v1.0.6?
If so, could you provide updated documentation or guidance on how to configure it?
Can test metrics and results be stored externally in databases such as Postgres or Delta Tables in the latest version?
If external storage is supported, what are the recommended methods or best practices to implement this?
If the metrics store has been deprecated, are there plans to provide alternative solutions or migrate existing users to new storage mechanisms?

**Impact:**
The ability to persist test metrics externally is crucial for our data analysis and reporting workflows. Clarification on this matter will help us decide whether to proceed with the upgrade or revert to the previous stable version until a suitable solution is available.

**Additional Information:**
Current Version: Great Expectations v1.0.6
Previous Version: Great Expectations v0.18
Environment: Running on Databricks using PySpark

**Related Discussion:**
[#10376](https://github.com/great-expectations/great_expectations/discussions/10376)

**Comment by OdyAsh** [comment]
Up
